src/publicApi.ts

```diff
@@ -320,7 +318,7 @@ export function rawWorkspaceToWorkspaceData(rawWorkspaces: Workspace | Workspace
             owner,
             repo,
             id: ws.workspaceId,
-            contextUrl,
+            contextUrl: url.toString(),
```

Comment:
it can be null as far as I understand, we will fail in such case here, I think we should adjust clients to be safe

Reply:
i.e. just filter such